spec/spec_helper.rb

```diff
@@ -14,9 +14,4 @@
     FileOperations.create_tmp_directory
     EnvPath.prepend_env_path!(TestPaths.fake_bin_path)
   end
-
-  config.before(:each, type: :feature) do |spec|
```

Comment:
This hook was needed for the end-to-end generator tests (the ones we just converted to pure generator tests) to properly clean up the `tmp` directory. Now that all tests are converted, we no longer need it! We can also remove `autoclean: false` from the remaining (real) feature specs.